templates/yum_distributor.json.erb

```diff
@@ -0,0 +1,8 @@
+{
+    <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::yum_remove_old_repodata_count']) -%>
+    "remove_old_repodata": True,
+    "remove_old_repodata_threshold": <%= scope['pulp::yum_remove_old_repodata_count'] %>
+    <% else %>
+    "remove_old_repodata": False
```

Comment:
is `False` valid JSON?

Reply:
good point, i was going by what was mentioned in this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1903367  

but it was later corrected, should be true